homeassistant/components/lektrico/strings.json

```diff
@@ -0,0 +1,101 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "description": "Set required parameters to connect to your device",
+        "data": {
+          "host": "[%key:common::config_flow::data::host%]",
+          "friendly_name": "[%key:common::config_flow::data::name%]"
+        }
+      },
+      "zeroconf_confirm": {
+        "description": "Do you want to add the Lektrico Charger with serial number `{serial_number}` to Home Assistant?",
+        "title": "Discovered Lektrico Charger device"
+      }
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+    },
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]",
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]"
+    }
+  },
+  "entity": {
+    "sensor": {
+      "state": {
+        "name": "State",
+        "state": {
+          "available": "Available",
+          "connected": "Connected",
+          "need_auth": "Connected,NeedAuth",
+          "paused": "Paused",
+          "charging": "Charging",
+          "error": "Error",
+          "updating_firmware": "Updating firmware"
+        }
+      },
+      "charging_time": {
+        "name": "Charging time"
+      },
+      "lifetime_energy": {
+        "name": "Lifetime energy"
+      },
+      "installation_current": {
+        "name": "Installation current"
+      },
+      "limit_reason": {
+        "name": "Limit reason",
+        "state": {
+          "no_limit": "No limit",
+          "installation_current": "Installation current",
+          "user_limit": "User limit",
+          "dynamic_limit": "Dynamic limit",
+          "schedule": "Schedule",
+          "em_offline": "EM offline",
+          "em": "EM",
+          "ocpp": "OCPP"
```

Comment:
What do these abbreviations mean?

Reply:
EM = Energy Meter
OCPP = Open Charge Point Protocol (it's the communication protocol between EV chargers and server)
Here, the sensor "Limit reason" shows who limited the charger's charging current.